workspaces/src/worker/impls.rs

```diff
@@ -174,16 +177,58 @@ impl Worker<Sandbox> {
         self.workspace.import_contract(id, worker)
     }
 
+    /// Patch state into the sandbox network, using builder pattern. This will allow us to set
+    /// state that we have acquired in some manner. This allows us to test random cases that
+    /// are hard to come up naturally as state evolves.
+    pub fn patch_state_builder(&self, account_id: &AccountId) -> SandboxPatchStateBuilder {
```

Comment:
Think it's better just to have one `patch_state` function that way we can be consistent and have a defacto way of doing patching state. Let's merge this one and the other and just have this *builder* like one be the default

Reply:
> Let's merge this one and the other

which 'other' one?

Reply:
I meant we can merge both patch_state and patch_state_builder to be just patch_state(AccountId) -> Builder. Would be a breaking API change, but it's fine since it would align with the other builder functions, and better to be consistent that way